boards/arm/mimxrt1050_evk/doc/index.rst

```diff
@@ -290,6 +290,10 @@ the `OpenSDA J-Link MIMXRT1050-EVK-Hyperflash Firmware`_. Check that jumpers
 J32 and J33 are **on** (they are on by default when boards ship from the
 factory) to ensure SWD signals are connected to the OpenSDA microcontroller.
 
+Follow the instructions in :ref:`Enable QSPI flash support in SEGGER JLink`_
```

Comment:
Don't use `:ref:` here, the `_` at the end links it.

Reply:
Fixed. Thank you for the heads up. I'm not too familiar with .rst file formatting.